src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  --main-blue-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    max-width: 924px;
```

Comment:
I think you'd better use 1024px
tip: you should also remember about **box-sizing** for rules about width math